man/rmdhunks/configuration.Rmd

```diff
@@ -0,0 +1,38 @@
+See also the [configuration vignette](https://docs.ropensci.org/vcr/articles/configuration.html).
+
+We set the following defaults:
+
+```{r, echo = FALSE, results='asis', collapse=TRUE}
```

Comment:
I replaced the hard-coded list with this, just to make sure it's easier to keep up-to-date.